minio/mcs
# [WIP] Connected add user-Groups to add user

## What does this do?

This PR connects the add user-groups API to the add user modal box

## How to Test

1. Create a new user using the add user module on mcs, keep the Inspetor opened in the network tab
2. Fill the information and save the user (remember the groups that you selected)
3. Check the network request & verify that the new user has the groups assigned in the groups request
4. Go to Groups tab & check each of the groups that you added to the user, the user must be selected in all of them
